llvm/lib/IR/DebugInfoMetadata.cpp

```diff
@@ -34,6 +35,12 @@ cl::opt<bool> EnableFSDiscriminator(
     cl::desc("Enable adding flow sensitive discriminators"));
 } // namespace llvm
 
+// When true, preserves line and column number by picking one of the merged
```

Comment:
I'd like to have a more descriptive name for this option, since we are discarding information, we're not preserving half of the merged source location. I came up with some suggestions:
* -merge-source-locs-by-picking
* -pick-merged-source-locations
* -merge-source-locations-with-selection
* -merge-source-locations-by-choice
* -no-approximate-merged-source-locs

I'm really just trying various synonyms for "pick", "select", "choose".

Reply:
Done, "pick-merged-source-locations" sounds good to me.